generator/webapp/src/main/java/io/fabric8/maven/generator/webapp/handler/OpenshiftWildFlyS2IHandler.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2016 Red Hat, Inc.
+ *
+ * Red Hat licenses this file to you under the Apache License, version
+ * 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package io.fabric8.maven.generator.webapp.handler;
+
+import io.fabric8.maven.core.config.PlatformMode;
+import io.fabric8.maven.core.util.MavenUtil;
+import org.apache.maven.project.MavenProject;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class OpenshiftWildFlyS2IHandler extends AbstractAppServerHandler {
```

Comment:
Not really sure about the name. What's the difference the WildFlyAppServerHandler ?